include/LiveScanClient/azureKinectCapture.h

```diff
@@ -17,16 +20,25 @@ class AzureKinectCapture : public ICapture
 	void MapColorFrameToCameraSpace(Point3f *pCameraSpacePoints);
 	void MapDepthFrameToColorSpace(UINT16 *pDepthInColorSpace);
 	void MapColorFrameToDepthSpace(RGB *pColorInDepthSpace);
+	k4a_image_t Downscale_image_2x2_binning(const k4a_image_t color_image);
+
 private:
 	k4a_device_t kinectSensor = NULL;
 	int32_t captureTimeoutMs = 1000;
 
 	k4a_image_t colorImage = NULL;
 	k4a_image_t depthImage = NULL;
-	k4a_image_t depthPointCloudImage = NULL;
+	k4a_image_t pointCloudImage = NULL;
+	k4a_image_t transformed_depth_image = NULL;
```

Comment:
This member, as well as some other ones, uses snake_case as opposed to camelCase used, elsewhere could you change it?